generators/app/index.js

```diff
@@ -4,111 +4,116 @@ const clor = require("clor")
 const yo = require("yosay")
 
 module.exports = yeoman.generators.Base.extend({
-  init: function () {
-    const done = this.async()
-
+  initializing: function() {
```

Comment:
So, small changes like `init` to `initializing` and `this.prompt...` to `prompting` are just changes in the Yeoman API?